src/pipecat/services/simli/video.py

```diff
@@ -118,8 +133,8 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
         """
         await super().process_frame(frame, direction)
         if isinstance(frame, StartFrame):
-            await self.push_frame(frame, direction)
             await self._start_connection()
+            await self.push_frame(frame, direction)
```

Comment:
Do we need this here now that we `push_frame` at the end of `process_frame`? Wondering if we're now pushing `StartFrame` twice.